### PR TITLE
"Search not found" responsive page

### LINKED ISSUE #1018: 'Search not found' page is not responsive in a mobile view when search for a long value
'Search not found' page is not responsive when search for a long value. As an example, when search for an invalid Tx hash/Order ID.
 See the image: 

Besides, the app logo has become center-aligned on such screen and is not visible until scroll the page

**Expected**: the page should be responsive in a mobile view.
App logo should be always visible.

### PR DESCRIPTION
**Pull request by alongoni**
# Summary

Closes #1018 
image

# To Test

1. Open the explorer with a [wrong address](https://pr1037--gpui.review.gnosisdev.com/search/0x69904ff6d6100799344E5C9A2806936318F6ba4fWRONG) in a mobile device or use the DevTools.

### CODE REVIEW

File: src/apps/explorer/styled.ts [hunk 1]
```diff
@@ -46,10 +45,7 @@ export const MainWrapper = styled.div`
   ${media.xSmallDown} {
     max-width: 100%;
     flex-grow: 1;
-    header {
-      margin-left: auto;
-      margin-right: auto;
-    }
+
```
Comment (henrypalacios) [inline src/apps/explorer/styled.ts:48]: I could have sworn that I have already done so in #1026

**Review by henrypalacios — APPROVED** [review]
Approved :+1:

**Comment by elena-zh** [comment]
Hey @alongoni , changes LGTM for a Mobile resolution. 
Still it would be nice to make screen's content look better in a tablet resolution with screen width in [740-920] px. Could you please fix it? 
https://watch.screencastify.com/v/Y3IDAwOdcnj2G6ZFpGaO

**Review by elena-zh — APPROVED** [review]
@alongoni , thanks! LGTM now. 
Btw, seems that #885 issue is also fixed here. I have linked it to the PR.

**Pull request merged**